src/App.js

```diff
@@ -9,7 +10,7 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article title={article.title} text={article.text} date={article.date} />
```

Comment:
You can use destructuring for article object before App component and write params value just - title={title}. 
Also in your case you need to write each params from tne new line) But with destructuring you can leave such notation, I think)